src/Features/CSharp/Portable/project.json

```diff
@@ -2,6 +2,8 @@
   "dependencies": {
   },
   "frameworks": {
-    ".NETPortable,Version=v4.5,Profile=Profile7": {}
+    "netstandard1.3": {
+      "imports": [ "portable-net45+win8", "dotnet" ]
```

Comment:
I think @jasonmalinowski removed multiple imports from everywhere to work around some issue.